packages/ripple-nuxt-tide/test/unit/__snapshots__/markup-transpiler.test.js.snap

```diff
@@ -3,12 +3,12 @@
 exports[`Markup transpiler should able transpile html into vue template with plugins 1`] = `
 "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
 <rpl-text-link :url=\\"linkData0.href\\" theme=\\"primary\\" :text=\\"linkData0.text\\"></rpl-text-link>
-<h4>Sort out your&apos;&gt;&lt;&amp;&apos;&quot;s finances</h4>
+<h4>Sort out your'&gt;&lt;&amp;'\\"s finances</h4>
```

Comment:
This is from the eslint upgrade. I need to review what causes the change and if this is the desired change.